src/stdlib/dns_lookup.rs

```diff
@@ -648,11 +696,10 @@ mod tests {
     }
 
     #[test]
-    fn test_custom_class_and_type() {
+    fn test_custom_class() {
         let result = execute_dns_lookup(DnsLookupFn {
-            value: expr!("localhost"),
-            class: expr!("*"),
```

Comment:
I couldn't get it working with `class` set to `*` so I just changed this test to verify the `type`.